### ISSUE TITLE
Activate #enable-force-dark flag in Chrome when Dark Mode becomes available

### ISSUE THREAD

**Issue by udobehm** [AutoDarkMode/Windows-Auto-Night-Mode#79]
Can you add the option to automatically enable the Chrome flag 'Force Dark Mode for Web Contents' (#enable-force-dark) once the dark mode in Windows gets activated? Would like to see every webpage in dark once the Windows dark scheme becomes available (of course disable that flag once Windows lights up again)...

**Comment by udobehm** [comment]
Hi Armin. This tutorial explains how Chrome can be set to always start in light or dark mode:
https://www.winhelponline.com/blog/chrome-always-start-light-or-dark-windows-10-app-mode/ (see registry edit below)...
Could you please 🧡 add an option to your app that it changes this registry entry always automatically when your app is configured to enter dark mode and when it enters the light mode??
I so much would like having this function as well as I want the Office 2019 theme change... Can you integrate this at least so far??

**Comment by Armin2208** [comment]
Chrome listens to the system theme reliable. What's your usecase of a manual force?

**Comment by udobehm** [comment]
Chrome only changes sites in a dark theme when the webmaster provides a dark theme as far as I know. The flag named ‘Force Dark Mode for Web Contents’ (#enable-force-dark) turn every page into a dark theme, no matter if the site provides a dark theme or not.
The problem is that this mode makes all web pages dark no matter what time of the day it is, whether the sun has risen or set or whether your app changes the system theme in Windows to light or dark theme.
So, basically, what I would like to have is to set an option in your app that allows to enable and disable this Chrome flag each time the dark or light theme in Windows becomes available. Means that all websites will appear in dark theme once the Windows theme is dark and all pages will appear in the normal appearance when the Windows light mode is on.

Von: Armin Osaj <notifications@github.com>
Gesendet: Freitag, 10. Januar 2020 10:18
An: Armin2208/Windows-Auto-Night-Mode <Windows-Auto-Night-Mode@noreply.github.com>
Cc: udobehm <udospostbox@outlook.de>; Author <author@noreply.github.com>
Betreff: Re: [Armin2208/Windows-Auto-Night-Mode] Activate #enable-force-dark flag in Chrome when Dark Mode becomes available (#79)


Chrome listens to the system theme reliable. What's your usecase of a manual force?

**Comment by udobehm** [comment]
Chrome only changes sites in a dark theme when the webmaster provides a dark theme as far as I know. The flag named ‘Force Dark Mode for Web Contents’ (#enable-force-dark) turn every page into a dark theme, no matter if the site provides a dark theme or not.
The problem is that this mode makes all web pages dark no matter what time of the day it is, whether the sun has risen or set or whether your app changes the system theme in Windows to light or dark theme.
So, basically, what I would like to have is to set an option in your app that allows to enable and disable this Chrome flag each time the dark or light theme in Windows becomes available. Means that all websites will appear in dark theme once the Windows theme is dark and all pages will appear in the normal appearance when the Windows light mode is on.

**Comment by Armin2208** [comment]
OK nice I will implement it for you ❤️

**Comment by udobehm** [comment]
Yeah 🤣🤣

**Comment by udobehm** [comment]
Hi Armin. Does it still need a lot more time to implement this feature??

**Comment by Armin2208** [comment]
Hello, I didn't worked on it yet. Was pretty busy.

**Comment by udobehm** [comment]
I see

**Comment by VincentJoshuaET** [comment]
Please also do it for Microsoft Edge, thanks

**Comment by udobehm** [comment]
Hi Armin, any news about this topic??

**Comment by Armin2208** [comment]
following problem: editing the registry key you mentioned above only changed the opening behavior if you click on a link in an app. But if you open chrome with your pinned taskbar shortcut it doesn't have any effect on that.

**Comment by Armin2208** [comment]
Hello! I really tried hard to make this possible. Sadly, my tests doesn't came to a good end.
The built in force dark mode in Chromium isn't working properly anyways, because many websites are glitching. Also it's hard to implement this for every single browser (chrome, edge, opera... and we need to left firefox alone).

This is why I link to the Dark Reader extension in Auto Dark Mode 3.0 (coming soon). [You can also visit the Wiki page](https://github.com/Armin2208/Windows-Auto-Night-Mode/wiki/Every-website-in-dark-mode-with-Dark-Reader). This extension colors every site in dark without any problems and it can follow the system theme set by Auto Dark Mode.

This is why I won't implement this issue. Sorry!

**Comment by udobehm** [comment]
Ok. Where do I find this extension? How is it called?

**Comment by Armin2208** [comment]
Dark Reader. You find it in every browser store. And I linked the wiki page in my comment.

**Comment by udobehm** [comment]
Ah, cool, thanks!

Von: Armin Osaj <notifications@github.com>
Gesendet: Samstag, 27. Juni 2020 22:48
An: Armin2208/Windows-Auto-Night-Mode <Windows-Auto-Night-Mode@noreply.github.com>
Cc: udobehm <udospostbox@outlook.de>; Author <author@noreply.github.com>
Betreff: Re: [Armin2208/Windows-Auto-Night-Mode] Activate #enable-force-dark flag in Chrome when Dark Mode becomes available (#79)


Dark Reader. You find it in every browser store. And I linked the wiki page in my comment.

**Comment by udobehm** [comment]
Can you quickly explain to me how the automatic switching works? So far it doesn’t seem like the Dark Reader extension changes its theme according to what is set in Windows… Thanks!

Von: Armin Osaj <notifications@github.com>
Gesendet: Samstag, 27. Juni 2020 22:48
An: Armin2208/Windows-Auto-Night-Mode <Windows-Auto-Night-Mode@noreply.github.com>
Cc: udobehm <udospostbox@outlook.de>; Author <author@noreply.github.com>
Betreff: Re: [Armin2208/Windows-Auto-Night-Mode] Activate #enable-force-dark flag in Chrome when Dark Mode becomes available (#79)


Dark Reader. You find it in every browser store. And I linked the wiki page in my comment.

**Comment by Armin2208** [comment]
Hi  there, I've updated the wiki page, so you can see how to set up Dark Reader: [Click here](https://github.com/Armin2208/Windows-Auto-Night-Mode/wiki/Dark-Mode-for-Webbrowser)

**Comment by Spiritreader** [comment]
Considering how specific this is, it seems to be the perfect job for a custom script; for more info see here:
https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/wiki/How-to-add-custom-scripts

Unless browser support is easier to toggle and without having to restart the browser it's unlikely to get added as ADM component.
